tests/features/media/entity_embed.feature

```diff
@@ -70,7 +70,7 @@ Feature: Embedding entities in a WYSIWYG editor
       | type | title  | path    |
       | page | Foobar | /foobar |
     When I visit "/foobar"
-    And I click "Edit"
+    And I click "Edit draft"
```

Comment:
The tab appears to be the same in HEAD. So doesn't look like anything has changed.